amqp_queue.c

```diff
@@ -445,7 +445,7 @@ static PHP_METHOD(amqp_queue_class, bind)
     if (PHP_AMQP_MAYBE_ERROR(res, channel_resource)) {
         php_amqp_zend_throw_exception_short(res, amqp_queue_exception_class_entry);
         php_amqp_maybe_release_buffers_on_channel(channel_resource->connection_resource, channel_resource);
-        return;
+        RETURN_THROWS();
```

Comment:
only cosmetic, add more assert in debug mode, make sense after each throw_exception